models/schemas/reviews.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const reviewSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    gameTitle: {
+        type: String,
+        required: true
+    },
+    shortDescription: {
+        type: String,
+        required: true,
+    },
+    review: {
+        type: String,
+        required: true
+    },
+    reviewerName: {
+        type: String,
+        required: true
+    },
+    dateReviewed: {
+        type: Date,
+        required: true
+    },
+    reviewerScore: {
+        type: String,
```

Comment:
Would a score be a string?

Reply:
We discussed that. We decided to keep it as a string so that we could write, for instance, `4/10`.

Reply:
Ah ok, cool